src/mlpack/methods/preprocess/preprocess_describe_main.cpp

```diff
@@ -0,0 +1,231 @@
+PARAM_INT_IN("dimension", "Dimension of the data. Use this to specify a "
+    "dimension", "d", 0);
+PARAM_INT_IN("precision", "Precision of the output statistics.", "p", 4);
+PARAM_INT_IN("width", "Width of the output table.", "w", 8);
+PARAM_FLAG("population", "If specified, the program will calculate statistics "
+    "assuming the dataset is the population. By default, the program will "
+    "assume the dataset as a sample.", "P");
+
+/**
+* Calculates the sum of deviations to the Nth Power
+*
+* @param input Vector that captures a dimension of a dataset
+* @param rowMean Mean of the given vector.
+* @return sum of nth power deviations
+*/
+double SumNthPowerDeviations(const arma::rowvec& input,
+    const double& rowMean,
+    const size_t Nth) // Degree of Power
+{
+  double sum = 0;
+  for (size_t i = 0; i < input.n_elem; ++i)
+  {
+    sum += pow(input(i) - rowMean, Nth);
+  }
+  return sum;
+}
+/**
+ * Calculates Skewness of the given vector.
+ *
+ * @param input Vector that captures a dimension of a dataset
+ * @param rowStd Standard Deviation of the given vector.
+ * @param rowMean Mean of the given vector.
+ * @return Skewness of the given vector.
+ */
+double Skewness(const arma::rowvec& input,
+    const double& rowStd,
+    const double& rowMean,
+    const bool population)
+{
+  double skewness = 0;
+  double S3 = pow(rowStd, 3);
+  double M3 = SumNthPowerDeviations(input, rowMean, 3);
+  double n = input.n_elem;
+  if (population)
+  {
+    // Calculate Population Skewness
+    skewness = n * M3 / (n * n * S3);
+  }
+  else
+  {
+    // Calculate Sample Skewness
+    skewness = n * M3 / ((n-1) * (n-2) * S3);
+  }
+  return skewness;
+}
+/**
+ * Calculates kurtosis of the given vector.
+ *
+ * @param input Vector that captures a dimension of a dataset
+ * @param rowStd Standard Deviation of the given vector.
+ * @param rowMean Mean of the given vector.
+ * @return Kurtosis of the given vector.
+ */
+double Kurtosis(const arma::rowvec& input,
+    const double& rowStd,
+    const double& rowMean,
+    const bool population)
+{
+  double kurtosis = 0;
+  double M4 = SumNthPowerDeviations(input, rowMean, 4);
+  double n = input.n_elem;
+  if (population)
+  {
+    // Calculate Population Excess Kurtosis
+    double M2 = SumNthPowerDeviations(input, rowMean, 2);
+    kurtosis = n * (M4 / pow(M2, 2)) - 3;
+  }
+  else
+  {
+    // Calculate Sample Excess Kurtosis
+    double S4 = pow(rowStd, 4);
+    double norm3 = (3 * (n-1) * (n-1)) / ((n-2) * (n-3));
+    double normC = (n * (n+1))/((n-1) * (n-2) * (n-3));
+    double normM = M4 / S4;
+    kurtosis = normC * normM - norm3;
+  }
+  return kurtosis;
+}
+/**
+ * Calculates standard error of standard deviation.
+ *
+ * @param input Vector that captures a dimension of a dataset
+ * @param rowStd Standard Deviation of the given vector.
+ * @return Standard error of the stanrdard devation of the given vector.
+ */
+double StandardError(const arma::rowvec& input, const double rowStd)
```

Comment:
I think we could replace rowvec by size_t, because this could reduce dependency on specific type.
In medium or large size project, reduce dependency of your api can make your codes easier to maintain.

ps : This is not a big problem in this small CLI program